varya/assets/sass/structure/_vertical-margins.scss

```diff
@@ -27,12 +27,20 @@
 }
 
 .site-header {
-	padding-top: calc(2 * var(--global--spacing-vertical));
-	padding-bottom: calc(2 * var(--global--spacing-vertical));
+	padding-top: calc(3 * var(--global--spacing-vertical));
+	padding-bottom: calc(3 * var(--global--spacing-vertical));
+
+	&.has-main-navigation {
```

Comment:
Moving `.has-main-navigation` to `<body>` means we’ll have to change these selectors to something like this. 

`.has-main-navigation & {}`